frontend/src/metabase/public/components/EmbedFrame/EmbedFrame.tsx

```diff
@@ -170,24 +187,45 @@ export const EmbedFrame = ({
       data-testid="embed-frame"
       data-embed-theme={theme}
     >
-      <ContentContainer>
+      <ContentContainer id={DASHBOARD_PDF_EXPORT_ROOT_ID}>
         {hasHeader && (
           <Header
-            className={EmbedFrameS.EmbedFrameHeader}
+            className={cx(
+              EmbedFrameS.EmbedFrameHeader,
+              SAVING_DOM_IMAGE_DISPLAY_NONE_CLASS,
```

Comment:
in the original implementation I added a new div with `id={DASHBOARD_PDF_EXPORT_ROOT_ID}` that was wrapping the parameters and the content, that broke some layouts though and the fix was to copy the styles/use ContentContainer also for the new div.
I found this solution to be a bit cleaner, I export everything in the ContentContainer but I hide the header, this should be more reliable to other style changes too